app/controllers/games_controller.rb

```diff
@@ -0,0 +1,68 @@
+class GamesController < ApplicationController
+  before_action :set_game, only: [:show, :edit, :update, :destroy]
+
+  # GET /games
+  # GET /games.json
+  def index
+    # @games = Game.all
+  end
+
+  # GET /games/1
+  # GET /games/1.json
+  def show
+  end
+
+  # GET /games/new
+  def new
+    @game = Game.new
+  end
+
+  # GET /games/1/edit
+  def edit
+  end
+
+  # POST /games
+  # POST /games.json
+  def create
+    @game = Game.new(game_params)
+
+    respond_to do |format|
+      if @game.save
+        format.html { redirect_to @game, notice: 'Game was successfully created.' }
+        format.json { render :show, status: :created, location: @game }
```

Comment:
I ❤️  that you put JSON responses in here - good forward thinking.

None of them are tested though *tisk tisk*.

```
get :create, :format => :json, ...
```